packages/amazonq/test/unit/amazonqGumby/transformationResultsHandler.test.ts

```diff
@@ -6,11 +6,19 @@ import assert from 'assert'
 import sinon from 'sinon'
 import os from 'os'
 import { DiffModel, AddedChangeNode, ModifiedChangeNode } from 'aws-core-vscode/codewhisperer/node'
+import { PatchInfo } from 'aws-core-vscode/codewhisperer'
 import path from 'path'
 import { getTestResourceFilePath } from './amazonQGumbyUtil'
 import { fs } from 'aws-core-vscode/shared'
 
 describe('DiffModel', function () {
+    let parsedTestDescription: PatchInfo
+    beforeEach(() => {
+        parsedTestDescription = JSON.parse(
+            '{"name": "Added file", "fileName": "resources/files/addedFile.diffs", "isSuccessful": true}'
```

Comment:
Typo with `diffs`? You can also run the test locally by going to the Run & Debug menu and instead click on "Run Extension Tests (current file)" with this file open to make sure they pass